docker-example/README.md

```diff
@@ -10,22 +28,48 @@ DASHBOARD_PASSWORD=0000000000000000000000000000000000000000000000000000000000000
 DASHBOARD_SALT=0000000000000000000000000000000000000000000000000000000000000000
 ```
 
-Run `./prepare_docker.sh` to create the `data` folder with correct permissions.
+Create the `data` folder with correct permissions by running the contained script:
 
-You can generate your own dashboard password and salt by running:
-`docker-compose run hornet tool pwd-hash`
+```
+./prepare_docker.sh
+```
 
-Change the values accordingly, then run `docker-compose up -d`
+Generate a password hash and salt for the password you want to use to access your dashboard:
 
-You can check the logs using `docker-compose logs`
+```
+docker compose run hornet tool pwd-hash
```

Comment:
docker-compose

Reply:
In the latest docker versions, docker compose is integrated and then you can use `docker compose`

Reply:
> In the latest docker versions, docker compose is integrated and then you can use `docker compose`

Interesting. A fresh ubuntu 20.04 doesn't have that functionality yet.

Reply:
https://docs.docker.com/engine/install/ubuntu/